projects/canopy/src/lib/button/button.notes.ts

```diff
@@ -16,23 +16,50 @@ Import the component in your module:
 
 and in your HTML:
 
-~~~
+~~~html
 <button lg-button type="button" [variant]="solid-primary">Button</button>
 ~~~
 
 or:
 
-~~~
+~~~html
 <a lg-button href="#" [variant]="primary">Link</a>
 ~~~
 
+## Button with icon with text
+
+Icons can be content projected into the button, the button component will then handle the styling of that icon.
+The \`ButtonIconPosition\` property can be used to locate the icon on the left hand side of the text, the default is to the right hand side.
+Content projection is used as Canopy Icons need loading via the iconRegistry which enables tree shaking of icons.
+
+~~~html
+<button lg-button type="button" [ButtonIconPosition]="left" >
+  Add item
+  <lg-icon name="add" />
+</button>
+~~~
+
+## Button with icon only
+
+Buttons can be setup as icon only buttons, this is done in a similar way to a button with icon and text.
+You will still need to include text content inside the button as this information is vital to screen readers.
+The \`iconButton\` property is used to visually hide that text from sighted users.
```

Comment:
Maybe drop the bit about sighted users, as there are many impediments that could play into it? Perhaps this bit could just be something like...

> The \`iconButton\` property is used to visually hide that text

Reply:
Good call.